modules/kernel/src/main/scala-2.10/notebook/ReplCalculator.scala

```diff
@@ -440,7 +440,7 @@ class ReplCalculator(
     val dummyScript = ("dummy", () => s"""val dummy = ();\n""")
     val SparkHookScript = (
       "class server",
-      () => s"""@transient val _5C4L4_N0T3800K_5P4RK_HOOK = "${repl.classServerUri.get}";\n"""
+      () => s"""@transient val _5C4L4_N0T3800K_5P4RK_HOOK = "${repl.classServerUri.get.replaceAll("\\\\", "\\\\\\\\")}";\n"""
```

Comment:
indeed, but the number of `\` is sensible to when who what is using the result (and how many times it's interpreted ^^). Hence, you'll see this with 2 -> 4 or 4 -> 8 like here.

anyway, if we can extract this logic as well, it would be awesome!